examples/bri-3/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "declaration": true,
+    "removeComments": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "es2017",
```

Comment:
i think we should go with highest possible target here, even es2022, is there a reason to go with es2017?

Reply:
This was default due to my version of node when setting up nestjs. Will bump as first next step.